library/core/src/ops/arith.rs

```diff
@@ -78,6 +78,13 @@ pub trait Add<Rhs = Self> {
     type Output;
 
     /// Performs the `+` operation.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// let x: u32 = 12;
+    /// assert_eq!(x + 1, 13);
```

Comment:
nit: this is more clear without the variable IMO
```suggestion
    /// assert_eq!(12 + 1, 13);
```